CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * [FEATURE] - [Live preview: Topic / Purpose](https://trello.com/c/JiSAAKBv/256-3-live-preview-topic-purpose)
 * [BUG] - [Researcher preview phone number/word missing](https://trello.com/c/Vt4JdtH5/293-bugs-researcher-preview)
 * [BUG] - [Live previews broken in every live environment](https://trello.com/c/BV2jgy9O/295-live-previews-broken-in-every-live-environment)
+* [FEATURE] - [Fix up typography styles](https://trello.com/c/CtYloVeU/286-fix-up-typography-styles)
```

Comment:
This PR doesn't feel like a `[FEATURE]` to me, but it also doesn't feel like a `[BUG]`.

@rgarner do you have any thoughts on this?

Reply:
@tosbourn Yes! The GDS take on this (with which I agree, largely) is that issues with people being able to read things is a `[BUG]`.

Reply:
I have rewrote the original ticket to be represented as a bug and will update the `CHANGELOG.md` update to be a `[BUG]`.